lib/match.rb

```diff
@@ -0,0 +1,147 @@
+require_relative "board"
+require_relative "player"
+
+class Match
+
+
+    def initialize(board)
+
+        @player_1 = Player.new("X")
```

Comment:
I advise against instantiating classes inside another class.

`Match` in this case knows too much about `Player`. One of the basic tenets of OOP (and its related principle [SOLID](https://subvisual.co/blog/posts/19-solid-principles-in-ruby/) is that we want our classes to be decoupled from one another. `Match` should only care about the fact that it uses a `player` object but doesn't have to care whether that `player` object comes from another class or from the planet Vulcan. 

What we did with `board` is the way to go here :)

Reply:
Thank you for the feedback. We will solve this.